README.md

```diff
@@ -0,0 +1,134 @@
+This is a client implementation of the Apache Camel Language Server Protocol for Emacs
```

Comment:
would be nice to provide a link for Apache Camel Language Server to the GitHub repository of the Apache Camel Language Server https://github.com/camel-tooling/camel-language-server/